cli/dcoscli/log.py

```diff
@@ -214,9 +218,34 @@ def dcos_log_enabled():
     :return: does cosmos have LOGGING capability.
     :rtype: bool
     """
-    return packagemanager.PackageManager(
+    has_capability = packagemanager.PackageManager(
         get_cosmos_url()).has_capability('LOGGING')
 
+    if not has_capability:
+        return False
+
+    base_url = config.get_config_val("core.dcos_url")
+    url = urllib.parse.urljoin(base_url, '/dcos-metadata/ui-config.json')
+
+    if not base_url:
+        raise config.missing_config_exception(['core.dcos_url'])
+
+    try:
+        response = http.get(url).json()
+    except (DCOSAuthenticationException, DCOSAuthorizationException):
+        raise
+    except DCOSException:
+        emitter.publish('Unable to determine if dcos-log is enabled')
```

Comment:
if the LOGGING capability exists, this endpoint should also exist, right?

Lets log the actual error and report What do you think about "Unable to determine logging mechanism for your cluster. Defaulting to X."?